webui/react/src/components/NavigationTabbar.tsx

```diff
@@ -51,6 +52,11 @@ const NavigationTabbar: React.FC = () => {
     setIsShowingOverflow(false);
   }, []);
 
+  const handlePathUpdate = useCallback((path) => {
+    history.push(path);
```

Comment:
suggestion: could we use handlePath from `routes/utils`?

Reply:
nice good call, I keep forgetting stuff like this lol

Reply:
that was quick! thx